rundeckapp/grails-app/jobs/rundeck/quartzjobs/ExecutionJob.groovy

```diff
@@ -281,6 +281,13 @@ class ExecutionJob implements InterruptableJob {
                     }
                     context.getScheduler().deleteJob(context.jobDetail.key)
                     return initMap
+                }else{
+                    //verify run on this node but scheduled disabled
+                    if(!initMap.scheduledExecution.scheduleEnabled){
```

Comment:
perhaps it should test `scheduledExecution.shouldScheduleExecution()` this will also fix it if the schedule is removed or the execution is disabled